data/presets/presets/amenity/bicycle_parking/shed.json

```diff
@@ -0,0 +1,33 @@
+{
+    "icon": "maki-bicycle",
+    "fields": [
+        "{amenity/bicycle_parking}",
+        "building"
+    ],
+    "geometry": [
+        "point",
+        "vertex",
+        "area"
+    ],
+    "tags": {
+        "amenity": "bicycle_parking",
+        "bicycle_parking": "shed"
+    },
+    "addTags": {
+        "building": "shed",
```

Comment:
So this is a problem because we don't want to add `building=shed` if the user maps it as a point.  This is why for presets that can be either points or areas, we include the `building_area` field.  It will set `building=yes` only if the thing is mapped as an area.

Reply:
Can you give me a hint what I need to change to make this preset add `building=shed + bicycle_parking=shed` if it is and area and only `bicycle_parking=shed` if it is a point?

Reply:
We can't change it right now (this is also #6007) .  So I think best to just leave `building` out of addTags and removeTags, and instead add a `building_area` field (which will set it to `building=yes` if it is an area).  This is the best we can do for right now and still not put `building` tags on points.

Reply:
Okay, done